internal/datacoord/compaction_trigger.go

```diff
@@ -119,8 +119,14 @@ func (t *compactionTrigger) start() {
 			case signal := <-t.signals:
 				switch {
 				case signal.isGlobal:
-					t.handleGlobalSignal(signal)
+					// ManualCompaction also use use handleGlobalSignal
+					// so throw err here
+					err := t.handleGlobalSignal(signal)
+					if err != nil {
+						log.Warn("unable to handleGlobalSignal", zap.Error(err))
```

Comment:
This seems to be no different from the error handling done before, as the previous approach to handling errors was simply to print them out.

Reply:
oh I see, it skiped the error in log messages. But maybe we can just add the error in log instead of returning an error

Reply:
return err because ManualCompaction use handleGlobalSignal here. Although there are some err, but when user invoke ManualCompaction, it still return success before. auto compaction is still the same.